### DISCUSSION TITLE
Add app icon?

### DISCUSSION THREAD

**Discussion by bnidevs** [fabiusBile/Spotify4BigSur#15] [General]
<img width="512" alt="mac1024" src="https://user-images.githubusercontent.com/33227410/160704082-18ef1f28-adfa-42b4-addf-8f293a24f45a.png">

I have attached an appiconset for use

[AppIcon.appiconset.zip](https://github.com/fabiusBile/Spotify4BigSur/files/8374910/AppIcon.appiconset.zip)

**Comment by fabiusBile** [comment]
Nice! I will add it in the next update!
